app/tests/spec/views/mixins/signup-disabled-mixin.js

```diff
@@ -7,20 +7,20 @@ define(function (require, exports, module) {
 
   var AuthErrors = require('lib/auth-errors');
   var chai = require('chai');
-  var SignupDisabledMixin = require('views/mixins/signup-disabled-mixin');
+  var SignUpDisabledMixin = require('views/mixins/signup-disabled-mixin');
```

Comment:
I noticed the weird file name on this test and fixed it up at the same time as I sorted out the capitalisation, hope that's cool.